nucypher/cli/options.py

```diff
@@ -12,14 +12,33 @@
 from nucypher.utilities.sandbox.middleware import MockRestMiddleware
 
 
-option_checksum_address = click.option(
-    '--checksum-address', help="Run with a specified account", type=EIP55_CHECKSUM_ADDRESS)
+# Alphabetical
+option_checksum_address = click.option('--checksum-address', help="Run with a specified account", type=EIP55_CHECKSUM_ADDRESS)
+option_config_file = click.option('--config-file', help="Path to configuration file", type=EXISTING_READABLE_FILE)
+option_config_root = click.option('--config-root', help="Custom configuration directory", type=click.Path())
+option_dev = click.option('--dev', '-d', help="Enable development mode", is_flag=True)
+option_db_filepath = click.option('--db-filepath', help="The database filepath to connect to", type=click.STRING)
+option_dry_run = click.option('--dry-run', '-x', help="Execute normally without actually starting the node", is_flag=True)
+option_etherscan = click.option('--etherscan/--no-etherscan', help="Enable/disable viewing TX in Etherscan")
+option_federated_only = click.option('--federated-only', '-F', help="Connect only to federated nodes", is_flag=True, default=None)
+option_force = click.option('--force', help="Don't ask for confirmation", is_flag=True)
+option_geth = click.option('--geth', '-G', help="Run using the built-in geth node", is_flag=True)
+option_hw_wallet = click.option('--hw-wallet/--no-hw-wallet')
+option_light = click.option('--light', help="Indicate that node is light", is_flag=True)
+option_m = click.option('--m', help="M-Threshold KFrags", type=click.INT)
+option_min_stake = click.option('--min-stake', help="The minimum stake the teacher must have to be a teacher", type=click.INT, default=0)
+option_n = click.option('--n', help="N-Total KFrags", type=click.INT)
+option_network = click.option('--network', help="Network Domain Name", type=click.STRING)
+option_poa = click.option('--poa', help="Inject POA middleware", is_flag=True, default=None)
+option_registry_filepath = click.option('--registry-filepath', help="Custom contract registry filepath", type=EXISTING_READABLE_FILE)
+option_staking_address = click.option('--staking-address', help="Address of a NuCypher staker", type=EIP55_CHECKSUM_ADDRESS)
+option_teacher_uri = click.option('--teacher', 'teacher_uri', help="An Ursula URI to start learning from (seednode)", type=click.STRING)
+_option_middleware = click.option('-Z', '--mock-networking', help="Use in-memory transport instead of networking", count=True)
 
-option_config_file = click.option(
-    '--config-file', help="Path to configuration file", type=EXISTING_READABLE_FILE)
 
-option_config_root = click.option(
-    '--config-root', help="Custom configuration directory", type=click.Path())
+#
+# Alphabetical
+#
 
 def option_controller_port(default=None):
```

Comment:
Do you want to intersperse these alphabetically with the other options above? It will help if any of the options above need to be parameterized later on (be it `required`, or different default value).

Reply:
Heh - that's how I found them I just didn't like the look of it.  What do you prefer?

Reply:
I feel like this whole common_options file is a hole that is getting very deep.

Reply:
There is a really good way to re-use common elements in object oriented programming.  This isn't it.

Reply:
No new additions here - for the scope of this PR - I'm just rearranging the declarations within the module.

Reply:
> Heh - that's how I found them I just didn't like the look of it. What do you prefer?

Haha. For me, keeping the entire thing in alphabetical order is preferred - but if you feel differently that's fine - I won't hold it against you 😆🤠